src/olympia/addons/forms.py

```diff
@@ -608,7 +608,7 @@ def save(self):
             'display_username': self.request.user.name
         }
         changed = False
-        for k, v in persona_data.iteritems():
+        for k, v in six.iteritems(persona_data):
```

Comment:
Why not simply do `persona_data.items()` if we don't necessarily care about performance implications here?